### PR TITLE
bookletcreator 2.1.0

### PR DESCRIPTION
**Pull request by unitof**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

---

Fails audit because this version was not added to [appcast](https://www.bookletcreator.com/update/bc-appcast.xml). Likely a mistake—I just notified the developer today. It is on the [changelog](https://www.bookletcreator.com/version-history/) and is the current version downloaded from the homepage.

### CODE REVIEW

**Comment by daeho-ro** [comment]
It supports arm now but the minimum version of macos is now `Mojave`, and the livecheck url is not updated for the most recent version.

**Comment by unitof** [comment]
Developer confirmed with me via email that Sparkle is no longer used after 2.0.5. Livecheck updated to alternative method.

File: Casks/b/bookletcreator.rb [hunk 1]
```diff
@@ -1,19 +1,20 @@
...
 
   url "https://www.bookletcreator.com/files/BookletCreator-#{version}.dmg"
   name "BookletCreator"
   desc "Booklet to PDF utility"
   homepage "https://www.bookletcreator.com/"
 
   livecheck do
-    url "https://www.bookletcreator.com/update/bc-appcast.xml"
-    strategy :sparkle, &:short_version
+    # 302 redirects to latest stable versioned download URL
```
Comment (bevanjkay) [inline Casks/b/bookletcreator.rb:11]:
```suggestion
```

We can remove the comment because the strategy is self-explanatory.

**Review by bevanjkay — APPROVED** [review]
Thank you @unitof

**Pull request merged**